static/js/components/ShiftPage.jsx

```diff
@@ -69,13 +69,13 @@ const ShiftPage = ({ route }) => {
             let usersHaveChanged = false;
             // check if the users have the same ids, or if they need a replacement when they didnt need one before, and vice versa
             for (let i = 0; i < updatedShift.shift_users.length; i += 1) {
+              const current_user = currentShift.shift_users[i];
+              const updated_user = updatedShift.shift_users[i];
```

Comment:
A bit everywhere on skyportal both backend and frontend we have a currentUser object. It's a bit confusing to use that name here. You could mistake it for the currentUser instead of the currentshiftuser. 
i would just call it `shift_user` and `updated_shift_user`, or simpler and maybe clearer: `previous_shift_user` / `new_shift_user`, or `old_shift_user` / `new_shift_user`.

Reply:
I chose `old_shift_user` / `new_shift_user`